Sources/Preferences/GlobalPreferences.swift

```diff
@@ -34,10 +34,82 @@ extension Preferences {
   }
   
   public final class Shields {
+    /// A 3 part option for shield levels varying in strength of blocking content
+    public enum ShieldLevel: String, CaseIterable {
```

Comment:
Move this type to `BraveShields`, doesn't belong in prefs

Reply:
If I do this I will have to make the settings public I'm trying to restrict access to this setting.

Reply:
The booleans are already public

Reply:
I was hiding them (making the private) so nobody uses them directly but I can make them private again.